include/NAS2D/Resources/Resource.h

```diff
@@ -24,9 +24,9 @@ namespace NAS2D {
 class Resource
 {
 public:
-	Resource();
+	Resource() = default;
 	Resource(const std::string& filePath);
-	virtual ~Resource();
+	virtual ~Resource() = 0;
```

Comment:
Now that I'm looking at this, this class seems kind of odd. It is an ADT. It has a pure virtual member function `load`, however the method is marked `private`. That means this class doesn't define a public interface. Rather it imposes implementation details.

We may need to look at this class more closely. It likely needs a new design.

As for the pure virtual destructor, that's mostly useful when there are no other member functions that can be marked as pure virtual. (Though, having an ADT with no public pure virtual methods is questionable in its own right). Instead, this class could have a defaulted destructor.

Reply:
I whole heartily agree that the Resource class isn't needed at all.

Reply:
Opened issue #401 to track further design review.